climada/hazard/test/test_tc_tracks.py

```diff
@@ -1359,6 +1359,111 @@ def test_track_land_params(self):
             on_land,
         )
 
+    def test_compute_density_tracks(self):
+        """Test `compute_track_density` to ensure proper density count."""
+        # create track
+        track = xr.Dataset(
+            {
+                "time_step": ("time", np.timedelta64(1, "h") * np.arange(4)),
+                "max_sustained_wind": ("time", [10, 20, 30, 20]),
+                "central_pressure": ("time", [1, 1, 1, 1]),
+                "radius_max_wind": ("time", [1, 1, 1, 1]),
+                "environnmental_pressure": ("time", [1, 1, 1, 1]),
+                "basin": ("time", ["NA", "NA", "NA", "NA"]),
+            },
+            coords={
+                "time": ("time", pd.date_range("2025-01-01", periods=4, freq="12H")),
+                "lat": ("time", [-90, -90, -90, -90]),
+                "lon": ("time", [-179, -169, -159, -149]),
+            },
+            attrs={
+                "max_sustained_wind_unit": "m/s",
+                "central_pressure_unit": "hPa",
+                "name": "storm_0",
+                "sid": "0",
+                "orig_event_flag": True,
+                "data_provider": "FAST",
+                "id_no": "0",
+                "category": "1",
+            },
+        )
+
+        tc_tracks = tc.TCTracks([track])
+
+        hist_abs, *_ = tc.compute_track_density(
+            tc_tracks,
+            res=10,
+            norm=None,
+        )
+        # hist_norm, *_ = tc.compute_track_density(tc_tracks, res=10, density=True)
+        hist_wind_min, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=11, wind_max=None
+        )
+        hist_wind_max, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=None, wind_max=30
+        )
+        hist_wind_max, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=None, wind_max=30
+        )
+        hist_wind_both, *_ = tc.compute_track_density(
+            tc_tracks, res=10, norm=None, wind_min=11, wind_max=29
+        )
```

Comment:
Please explain in comments what are the values that should come out so that , in case the test fails in the future, a reader can immediately understand whether the new values are to be expected or are wrong.